src/main/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/impl/BalanceSheetTransformerImpl.java

```diff
@@ -142,6 +211,63 @@ private void populatePreviousFixedAssets(BalanceSheet balanceSheet, FixedAssetsA
         }
     }
 
+    private void populatePreviousCurrentAssets(BalanceSheet balanceSheet, CurrentAssetsApi currentAssetsApi) {
+
+        CurrentAssets currentAssets = createCurrentAssets(balanceSheet);
+
+        // Stocks
+        if (currentAssetsApi.getStocks() != null) {
+            Stocks stocks = createStocks(balanceSheet);
+            stocks.setPreviousAmount(currentAssetsApi.getStocks());
+        }
+
+        // Debtors
+        if (currentAssetsApi.getStocks() != null) {
+            Debtors debtors = createDebtors(balanceSheet);
+            debtors.setPreviousAmount(currentAssetsApi.getDebtors());
+        }
+
+        // Cash at bank and in hand
+        if (currentAssetsApi.getCashAtBankAndInHand() != null) {
+            CashAtBankAndInHand cashAtBankAndInHand = createCashAtBankAndInHand(balanceSheet);
+            cashAtBankAndInHand.setPreviousAmount(currentAssetsApi.getCashAtBankAndInHand());
+        }
+
+        // Total
+        if (currentAssetsApi.getTotal() != null) {
+            currentAssets.setPreviousCurrentAssetsTotal(currentAssetsApi.getTotal());
+        }
+    }
+
+    private void populateCurrentCurrentAssets(BalanceSheet balanceSheet, CurrentAssetsApi currentAssetsApi) {
```

Comment:
Could we maybe rename this as something like `populateCurrentPeriodCurrentAssets` for clarity? If we chose to do this, it would also be worth renaming the other methods to make them consistent